cardano-lib/generic-tracer-config.nix

```diff
@@ -0,0 +1,53 @@
+{
+  # Cardano-tracer will use default values where parameters are set to null.
+  WarnRTViewMissing = null;
+  ekgRequestFreq = null;
+  ekgRequestFull = null;
+
+  # Whether to enable an EKG http interface for process monitoring.
+  hasEKG = {
+    epHost = "127.0.0.1";
+    epPort = 12788;
+  };
+
+  # Whether to enable a prometheus export of metrics.
+  hasPrometheus = {
+    epHost = "127.0.0.1";
+    epPort = 12808;
+  };
+
+  loRequestNum = null;
```

Comment:
Maybe add comment to clarify:
`# How many trace objects will be forwarded with each request`

Reply:
Good idea.  I have fairly detailed descriptions for everything in the nixos tracer service in the node PR, so I have ported all those descriptions over to this generic tracer config with a few tweaks here and there.  I think these new verbose descriptions should cover everything you have suggested in the comments here, so I will mark these comments as resolved, but please feel free to re-flag anything you think is still an issue.